src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/config/ServiceUrlHelper.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.notificationsalertsvsip.config
+
+import org.springframework.stereotype.Component
+
+@Component
+class ServiceUrlHelper {
+  val firstPartUrl = "https://hmpps-notifi-alerts-vsip"
```

Comment:
is it notifi or notifications?

Reply:
Thats the url name, for the server

Reply:
It's this https://hmpps-notifi-alerts-vsip-dev.prison.service.justice.gov.uk